.github/workflows/deploy.yml

```diff
@@ -130,10 +130,7 @@ jobs:
           CONFIRM_PRODUCTION: true
 
       - name: Terraform init, plan & apply
-        run: |
-          terraform init -backend-config=workspace_variables/${{ matrix.environment }}_backend.tfvars
-          terraform plan -var-file workspace_variables/${{ matrix.environment }}.tfvars.json -out tfplan
-          terraform apply -auto-approve -input=false "tfplan"
+        run: make ${{ matrix.environment }} ci deploy
```

Comment:
TF_VAR_paas_docker_image would be overwritten by "main" if IMAGE_TAG is not passed

Should we set the IMAGE_TAG env var to the commit SHA as per https://github.com/DFE-Digital/register-trainee-teachers/blob/main/.github/workflows/deploy.yml#L120 instead?

Reply:
I've updated the env vars to include IMAGE_TAG